code/obj/item/stun_baton.dm

```diff
@@ -232,10 +232,10 @@
 						user.show_text("The [src.name] is out of charge!", "red")
 				return
 
-			if ("failed_harm")
-				user.visible_message("<span class='alert'><B>[user] has attempted to beat [victim] with the [src.name] but held it wrong!</B></span>")
-				playsound(get_turf(src), "sound/impact_sounds/Generic_Stab_1.ogg", 50, 1, -1)
-				logTheThing("combat", user, victim, "unsuccessfully tries to beat %target% with the [src.name] at [log_loc(victim)].")
+//			if ("failed_harm")
```

Comment:
IMO just remove these lines instead of commenting out, version control exists to keep source history

Reply:
fair, but i asked mbc about this and he said it was ok. theres tons of legacy features commented out in code already. shrug.